apps/tup-ui/src/main.tsx

```diff
@@ -3,7 +3,8 @@ import { QueryClient, QueryClientProvider } from 'react-query';
 import { BrowserRouter } from 'react-router-dom';
 import ReactDOM from 'react-dom/client';
 import App from './App';
-//import './index.css';
+
+import './main.css';
```

Comment:
ℹ️ I found a use for [this stylesheet](https://github.com/TACC/tup-ui/pull/47/files#diff-d3a034d2bd8e0eb8ffb07a28f730fce5185cfe7407059ffe2621fecd02172eb9) (and renamed from `index.css` to `main.css` to match `main.jsx`).